plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/navigator/fs/DBNPathBase.java

```diff
@@ -288,7 +289,18 @@ public void dropNodes(DBRProgressMonitor monitor, Collection<DBNNode> nodes) thr
                     break;
                 }
                 Path resource = node.getAdapter(Path.class);
-                if (resource == null || !Files.exists(resource)) {
+                if (resource == null) {
+                    monitor.subTask("Copy file");
+                    try (InputStream inputStream = node.getAdapter(InputStream.class)) {
```

Comment:
you will log in momitor even if input stream is null.
Check for null before any operations/logging